samples/drivers/watchdog/Makefile

```diff
@@ -1,4 +1,12 @@
 BOARD ?= quark_d2000_crb
-CONF_FILE = prj.conf
+
+ifeq ($(BOARD),96b_carbon)
+CONF_FILE ?= prj_$(BOARD).conf
+else
+ifeq ($(BOARD),nucleo_f334r8)
+CONF_FILE ?= prj_$(BOARD).conf
```

Comment:
This is nice, do you think we can use CONFIG_SOC_SERIES_STM32_... instead of board names to make it easily usable for other boards?